ghcide/src/Development/IDE/GHC/ExactPrint.hs

```diff
@@ -67,11 +67,6 @@ import Data.Monoid (All(All))
 #if __GLASGOW_HASKELL__ == 808
 import Control.Arrow
 #endif
-#if __GLASGOW_HASKELL__ > 808
-import Bag (listToBag)
-import ErrUtils (mkErrMsg)
-import Outputable (text, neverQualify)
-#endif
```

Comment:
Why did this `#if` disappear? Seems unrelated.

Reply:
doh, again! It's the second time it happens to me, I think this could be a problem with the pre-commit hook, /cc @Ailrun

Reply:
Could be... It's probably because CPP support for the formatter is not so reliable. Unfortunately, I don't know any formatter works well with CPP...